stdlib/ballerina-builtin/src/main/ballerina/ballerina/jwt/issuer.bal

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package ballerina.jwt;
+
+import ballerina.util;
+import ballerina.jwt.signature;
+
+@Description {value:"Represents JWT issuer configurations"}
+public struct JWTIssuerConfig {
+    string certificateAlias;
+    string keyPassword;
+}
+
+@Description {value:"Issue a JWT token"}
+@Param {value:"header: Header object"}
+@Param {value:"payload: Payload object"}
+@Param {value:"config: JWTIssuerConfig object"}
+@Return {value:"string: JWT token string"}
+@Return {value:"error: If token validation fails "}
+public function issue (Header header, Payload payload, JWTIssuerConfig config) (string, error) {
```

Comment:
Shouldn't  we add the jti value here as well?

Reply:
You can set that within payload

Reply:
:+1: